### ISSUE TITLE
TikZ rendering problems

### ISSUE THREAD

**Issue by Alex-Corner** [chirun-ncl/chirun#232]
Something else has happened in a recent update to rendering, causing TikZ pictures to throw errors on building. All of my diagrams built two weeks ago, but I can't seem to find in the recent pushes what might have caused these problems in the last week.

The Build Log has this error message: `CalledProcessError: Command 'pdflatex images-18cojh87.tex' returned non-zero exit status 1.`

In the Detailed Log there is also `ERROR: Loading package "tikz" raised exception ImportError : cannot import
   name 'VerbatimEnvironment' from 'chirun.plastex'
   (/usr/local/lib/python3.10/dist-packages/chirun/plastex/__init__.py)` whenever \usepackage{tikz} appears in the preamble.

Commenting out all of the TikZ diagrams, but leaving one to cause the error, starts to run into problems as soon as it hits that diagram. The log from that point on is attached as a .txt file. [error_log_240409.txt](https://github.com/chirun-ncl/chirun/files/14917622/error_log_240409.txt)
`

**Comment by christianp** [comment]
Yeah, sorry, I just reverted the commit that did that, so hopefully it's fixed for now. I'll be able to do things properly after this session at BAMC!

**Comment by Alex-Corner** [comment]
Brilliant, thank you! Seems to go through fine now. Enjoy the BAMC!
